test/Datadog.Trace.ClrProfiler.IntegrationTests/SqlServerTests.cs

```diff
@@ -30,8 +30,8 @@ public void SubmitsTraces()
                 Assert.True(spans.Count > 0, "expected at least one span");
                 foreach (var span in spans)
                 {
-                    Assert.Equal(SqlServerIntegration.OperationName, span.Name);
-                    Assert.Equal($"Samples.SqlServer-{SqlServerIntegration.ServiceName}", span.Service);
+                    Assert.Equal("sql-server.query", span.Name);
```

Comment:
Why do we not also have integration tests for postgres?

Reply:
The short answer is "time." Postgres support is time-sensitive. I tested Postgres manually and the SQL Server tests cover ADO.NET as well. I'm adding more automated integration tests for different ADO.NET providers in a separate PR.